packages/sitecore-jss-dev-tools/src/templating/utils.ts

```diff
@@ -149,7 +95,7 @@ export function writePlugins(listPath: string, rootPath: string, moduleType: Mod
  * @param {string} path path to get plugin from
  * @param {string} pluginName plugin name
  */
-function getPluginList(path: string, pluginName: string): PluginFile[] {
+export function getPluginList(path: string, pluginName: string): PluginFile[] {
```

Comment:
Needs to be covered by unit tests, better to check coverage for the full picture